package.json

```diff
@@ -1,5 +1,9 @@
 {
-  "dependencies": {
```

Comment:
Why is this removed for linting?

Reply:
This is unwanted code added long ago. We have this dependency in `server/package.json` file. This dependency do nothing at the root directory.